application/models/Token.php

```diff
@@ -438,7 +438,12 @@ public function rules()
             $aRules[] = array(
                 $key, 'filter',
                 'filter' => array(self::class, 'sanitizeAttribute'),
-                'except' => 'FinalSubmit'
+                'on' => 'register'
+            );
+            $aRules[] = array(
+                $key,
+                'LSYii_Validators',
+                'except' => 'finalSubmit,register'
```

Comment:
XSS filter needs also to be applied for admins according to XSS security setting, otherwise a malicious admin can embed XSS data into a participant attribute and XSS another admin for access.

Reply:
Ignore my comment - I misread.

Reply:
I can force xss protection for superamin too, we have tools for this.